Dockerfile-README.md

```diff
@@ -0,0 +1,55 @@
+# Dockerfile README
+
+This document specifies how **Dockerfile** should be written and setup for all Elevate micro-services.
+
+## Specifications
+
+1.  Location: **_Root directory of the service._**
+2.  File name: **_Dockerfile_**
+
+<br>
+
+## Expected Root Directory Structure
+
+    ./notification/
```

Comment:
@joffinjoy it should be /documents/

Reply:
@rakeshSgr  Assuming document service wouldn't be an apt representation of usual Elevate services, setting the service name to more generic <service-name>.